styleguide/source/_patterns/01-molecules/06-components/22-article-preview/article-preview~small-image.json

```diff
@@ -1,6 +1,6 @@
 {
   "topic_article": {
-    "image": "https://ipsumimage.appspot.com/279x186?l=3:2|279x186&s=36",
+    "image": "https://ipsumimage.appspot.com/450x321?l=7:5|450x321&s=36",
```

Comment:
This changes the image correctly, however in the twig file (https://github.com/AmericanMedicalAssociation/AMA-style-guide/blob/develop/styleguide/source/_patterns/01-molecules/06-components/22-article-preview/article-preview.twig#L11) isn't changing the image based on variant.